test/unit/runner.spec.js

```diff
@@ -454,7 +454,7 @@ describe('Runner', function() {
 
     describe('shortStackTrace', function() {
       beforeEach(function() {
-        if (path.sep !== '/') {
+        if (process.platform === 'win32') {
```

Comment:
It looks like the failed tests weren't meant to run because `path.sep !== '/'` rules out browsers, but `process.platform == 'win32'` doesn't. Haven't investigate how this unit test runs in browser though. It seems `requrie('path')` just gives empty obejct `{}`.

Reply:
@boneskull It's because `path` and `fs` are both ignored in the browserify config from `karma.conf.js`. IMHO, several changes are needed to fix this unit test:

- calibrate test data `stack` by replacing `/` to `path.sep` if `path.sep === '\'`
- change the way multiline error message is generated in browser, using `document.body.outerHTML` perhaps? Or just remove this test instead, which is just complimentary to the single line test anyway. The issue is already covered.

Reply:
this test probably doesn’t belong lumped in with the unit tests, and/or the code under test should be refactored out of the runner. the unit tests _shouldn’t_ be platform-dependent.

good catch

Reply:
> It looks like the failed tests weren't meant to run because `path.sep !== '/'` rules out browsers, but `process.platform == 'win32'` doesn't. Haven't investigate how this unit test runs in browser though. It seems `requrie('path')` just gives empty obejct `{}`.

Somewhere in browserify-land, `path.sep` is hardcoded to use POSIX. `process.platform` [won't exist](https://github.com/defunctzombie/node-process/blob/master/browser.js) when browserified.